src/discussion/llm-implementation.tex

```diff
@@ -44,3 +44,17 @@ \section{Large language model implementation}\label{sec:large-language-model-imp
 not able to contribute to the learning experience as much as the team had hoped.
 Reflecting on this, the team believes that an LLM could be a valuable addition to the project, and should be implemented
 if the project were to be expanded.
+
+% textidote: ignore begin
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=0.85\textwidth]{Discussion-game-chat}
+    \caption{What the chat could look like with an LLM.}\label{fig:Discussion-game-chat}
+\end{figure}
+% textidote: ignore end
+
+The purpose of this figure is to show what it could look like if we had implemented an LLM in the project, one such as
```

Comment:
It is not clear what "this picture" refers to, as both pictures are only referenced en later sentences.